src/cc_catalog_airflow/dags/provider_api_scripts/metropolitan_museum_of_art.py

```diff
@@ -184,19 +169,12 @@ def _get_data_for_each_image(object_id):
         foreign_url,  # foreign url of image
         image_url,  # image url
         thumbnail,  # thubnail url
-        None,  # license URl
         'cc0',  # license
         '1.0',  # license verion
         foreign_id,  # foreign identifier
-        None,  # width
-        None,  # height
         creator_name,  # creator name
-        None,  # creator url
         title,  # title
         meta_data,  # meta data
```

Comment:
The problem now is that you've removed the extra `None`s but not changed to *keyword* arguments.  This will not work.  You need to change this.  So, instead of
```python
         foreign_url,  # foreign url of image 
```
you need
```python
        foreign_landing_url=foreign_url, # foreign url of image
```

Reply:
This is for *all* arguments of `image_store.add_item`